spec/views/posts/show_spec.rb

```diff
@@ -42,12 +41,10 @@
   it 'shows the comments on the post users left lik(Thank you for your support 😊)' do
     expect(page).to have_content(@com1.text)
     expect(page).to have_content(@com2.text)
-    expect(page).to have_content(@com3.text)
   end
 
   it 'shows the user name of a comment' do

```

Comment:
- [ ] @team kindly change the title of this test to match what it's testing, displaying the commentator name